header.php

```diff
@@ -16,8 +16,8 @@
 
   <body <?php body_class(); ?>>
     
-    <section id='header'>
+    <header class="site-header">
```

Comment:
Nice. (Single quote should suffice but otherwise great moving from ID to classes)